__tests__/src/core/game.spec.js

```diff
@@ -0,0 +1,190 @@
+import * as Game from '../../../src/core/game';
+import Coords from '../../../src/core/coords';
+
+describe('Game', () => {
+    describe('buildGrid', () => {
+        test('should build a grid of size 3', () => {
+            const grid = Game.buildGrid(3);
+            const expectedGrid = [[1, 2, 3], [4, 5, 6], [7, 8, 0]];
+
+            expect(grid.length).toEqual(3);
+            expect(grid).toEqual(expectedGrid);
+        });
+
+        test('should build a grid of size 4 by default', () => {
+            const expectedGrid = [
+                [1, 2, 3, 4],
+                [5, 6, 7, 8],
+                [9, 10, 11, 12],
+                [13, 14, 15, 0],
+            ];
+
+            const grid = Game.buildGrid();
+
+            expect(grid.length).toEqual(4);
+            expect(grid).toEqual(expectedGrid);
+        });
+    });
+
+    describe('deepCopyGrid', () => {
+        test('should copy a grid', () => {
```

Comment:
How do you know it's not the plain same grid and that it's been actually copied ?